utils/docker/Dockerfile.leap.15

```diff
@@ -86,71 +103,57 @@ RUN zypper --non-interactive update && \
-                    build_number="${branch#*:}";                                                                 \
-                    branch="${branch%:*}";                                                                       \
-                fi;                                                                                              \
-            fi;                                                                                                  \
-            zypper --non-interactive addrepo --no-gpgcheck                                                       \
-                   ${JENKINS_URL}job/daos-stack/job/$repo/job/$branch/$build_number/artifact/artifacts/leap15/   \
-                   $repo:$branch:$build_number;                                                                  \
-        done;                                                                                                    \
-        echo "Installing: $QUICKBUILD_DEPS";                                                                     \
-        echo "$QUICKBUILD_DEPS" | sed -e '/^$/d' | tr '\n' '\0' | xargs -0 zypper --non-interactive install;     \
+RUN if $QUICKBUILD; then                                                       \
+        for repo in $REPOS; do                                                 \
+            branch="master";                                                   \
+            build_number="lastSuccessfulBuild";                                \
+            if [[ $repo = *@* ]]; then                                         \
+                branch="${repo#*@}";                                           \
+                repo="${repo%@*}";                                             \
+                if [[ $branch = *:* ]]; then                                   \
+                    build_number="${branch#*:}";                               \
+                    branch="${branch%:*}";                                     \
+                fi;                                                            \
+            fi;                                                                \
+            zypper --non-interactive addrepo --no-gpgcheck                     \
+                   ${JENKINS_URL}job/daos-stack/job/$repo/job/$branch/$build_number/artifact/artifacts/leap15/ \
+                   $repo:$branch:$build_number;                                \
+        done;                                                                  \
+        zypper --non-interactive --gpg-auto-import-keys --no-gpg-checks ref;   \
+        echo "Installing: $QUICKBUILD_DEPS";                                   \
+        echo "$QUICKBUILD_DEPS" | sed -e '/^$/d' | tr '\n' '\0' |              \
+              xargs -0 zypper --non-interactive install;                       \
+        zypper clean --all;                                                    \
```

Comment:
Earlier in this file you called "zypper --non-interactive clean --all", is the --non-interactive part needed or not?

Reply:
The --non-interactive apparently is not needed,
Fixing the dockerfile to be consistent.